ui/page/settings/view.jsx

```diff
@@ -262,7 +277,17 @@ class SettingsPage extends React.PureComponent<Props, State> {
     const endHours = ['5', '6', '7', '8'];
 
     return (
-      <Page className="card-stack">
+      <Page
+        noFooter
+        noSideNavigation
+        backout={{
+          backFunction: () => this.onDone(),
+          title: __('Settings'),
+          simpleTitle: __('Editing'),
```

Comment:
I should probably just remove `simpleTitle`, I'm not a fan of it. It is supposed to be used for mobile since "Editing @channelName` won't fit on most phone screens. In this case we don't need one. It should just use the regular title if no `simpletitle` is passed.